sale_procurement_amendment/models/sale_order_line.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+from odoo.tools import float_compare
+
+
+class SaleOrderLine(models.Model):
+
+    _inherit = 'sale.order.line'
+
+    to_be_procured = fields.Boolean(
+        compute='_compute_to_be_procured',
+        readonly=True,
```

Comment:
@rousseldenis remove it, compute always `readonly`

Reply:
yes